api/src/pcapi/routes/backoffice/accounts/blueprint.py

```diff
@@ -86,6 +86,10 @@ def search_public_accounts() -> utils.BackofficeResponse:
             form.add_error_to(typing.cast(str, error["loc"][0]))
         return render_search_template(form), 400
 
+    if len(search_model.terms) <= 3 and not str(search_model.terms).isnumeric():
+        flash("Attention la recherche doit contenir au moins 3 lettres.", "warning")
```

Comment:
```suggestion
        flash("Attention, la recherche doit contenir plus de 3 lettres.", "warning")
```
Juste une suggestion

Reply:
Hmmm, plus de 3 lettres ça voudrait dire au moins 4, donc je préfère « au moins 3 »

Reply:
Juste une question, pourquoi ne pas plutôt mettre cela dans le validateur wtforms ?

Reply:
1) erreur de ma part on avait dit qu'on aceptait 3 lettres (je corrigerai lundi)

2) la validation est faite dans pydantic. ce serait faisable mais je ne pense pas que le message d'erreur serait tres élégant(mais j'ai pas essayé plus que ça)

Reply:
La validation pydantic après celle de wtforms mériterait d'être supprimée dans un chantier technique, c'est tellement bizarre de faire ainsi.
J'aurais vu ta vérification dans la méthode `validate_terms` qui existe déjà dans le formulaire pour vérifier l'absence de %, avec un message d'erreur que tu choisis :
https://github.com/pass-culture/pass-culture-main/blob/master/api/src/pcapi/routes/backoffice/forms/search.py#L45

Reply:
done